data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/extension/DefaultKafkaClusterConfigSupplier.java

```diff
@@ -19,21 +20,21 @@ public DefaultKafkaClusterConfigSupplier(KafkaClusterConfig kafkaClusterConfig)
 
     @Override
     public List<String> getBootStrapServers() {
-        return kafkaClusterConfig.getBootStrapServers();
+        return Objects.nonNull(kafkaClusterConfig) ? kafkaClusterConfig.getBootStrapServers() : null;
```

Comment:
Is the Kafka source handling when these values are null ?

Reply:
How about retruning empty list

Reply:
@dinujoh , I agree with @hshardeesi  here. Returning an empty list would be better.